topo/node/cisco/cisco.go

```diff
@@ -198,18 +198,92 @@ func fmtInt400(eid int) string {
 	return fmt.Sprintf("FourHundredGigE0/0/0/%d", eid)
 }
 
-func getCiscoInterfaceId(pb *tpb.Node, eth string) (string, error) {
-	ethWithIdRegx := regexp.MustCompile(`e(t(h(e(r(n(e(t)*)*)*)*)*)*)\d+`) // check for e|et|eth|....
+func setE8000Env(pb *tpb.Node) error {
+	interfaceList := "MgmtEther0/RP0/CPU0/0"
+	interfaceMap := "MgmtEther0/RP0/CPU0/0:eth0"
+	eths := make([]string, 0)
+	for k := range pb.Interfaces {
+		eths = append(eths, k)
+	}
+	sort.Strings(eths)
+	for _, eth := range eths {
+		ciscoInterfaceID, err := getCiscoInterfaceID(pb, eth)
+		if err == nil {
```

Comment:
more common pattern is to check

if err != nil {
  return err
}
interfaceList = ...

Reply:
done